lib/broadway_rabbitmq/producer.ex

```diff
@@ -271,6 +271,66 @@ defmodule BroadwayRabbitMQ.Producer do
   One way to handle this is by using [Dead Letter Exchanges](https://www.rabbitmq.com/dlx.html)
   and [TTL and Expiration](https://www.rabbitmq.com/ttl.html).
 
+  ### Dead letter exchange
+
+  #### Example
+
+      use Broadway
+
+      @exchange "x.my_exchange"
+      @exchange_dlx "x.my_exchange.dlx
+      @queue "x.my_queue"
+      @queue_dlx "x.my_queue.dlx"
+
+      def start_link(_opts) do
+        initialize_queues()
```

Comment:
You shouldn't do this in here, you should use the `:after_connect` option (see the docuemntation)